src/js/media-chrome-button.ts

```diff
@@ -331,6 +339,30 @@ class MediaChromeButton extends globalThis.HTMLElement {
     setStringAttr(this, Attributes.TOOLTIP_PLACEMENT, value);
   }
 
+  get mediaController(): string | undefined {
+    return getStringAttr(this, MediaStateReceiverAttributes.MEDIA_CONTROLLER);
+  }
+
+  set mediaController(value: string | undefined) {
+    setStringAttr(this, MediaStateReceiverAttributes.MEDIA_CONTROLLER, value);
+  }
+
+  get disabled(): boolean | undefined {
+    return getBooleanAttr(this, Attributes.DISABLED);
+  }
+
+  set disabled(value: boolean | undefined) {
+    setBooleanAttr(this, Attributes.DISABLED, value);
+  }
+
+  get noTooltip(): boolean | undefined {
+    return getBooleanAttr(this, Attributes.NO_TOOLTIP);
+  }
+
+  set noTooltip(value: boolean | undefined) {
+    setBooleanAttr(this, Attributes.DISABLED, value);
```

Comment:
```suggestion
    setBooleanAttr(this, Attributes.NO_TOOLTIP, value);
```

a good habit to get into is self-reviewing 1 time before you commit, I personally do at the time of git staging. just a quick glance to the diff to detect any little mistakes.